Julia template generation error after changing jdk

When moving to a new computer I started to get 

```
java -Xmx2048m -jar LmcpGen.jar -mdmdir ~/MDMs/ -jl -dir ./output
Exception in thread "main" java.lang.IllegalArgumentException: java.lang.Exception: Method calling exception.  Trying to access method name: root_module_includes for file: LMCP.jl
        at avtas.lmcp.lmcpgen.LmcpGen.main(LmcpGen.java:378)
        at avtas.lmcp.lmcpgen.LmcpGenGUI.main(LmcpGenGUI.java:222)
Caused by: java.lang.Exception: Method calling exception.  Trying to access method name: root_module_includes for file: LMCP.jl
        at avtas.lmcp.lmcpgen.LmcpGen.replaceTags(LmcpGen.java:221)
        at avtas.lmcp.lmcpgen.LmcpGen.makePackage(LmcpGen.java:72)
        at avtas.lmcp.lmcpgen.LmcpGen.main(LmcpGen.java:375)
        ... 1 more
Caused by: java.lang.reflect.InvocationTargetException
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:498)
        at avtas.lmcp.lmcpgen.LmcpGen.replaceTags(LmcpGen.java:217)
        ... 3 more
Caused by: java.lang.NoSuchMethodError: java.lang.String.isBlank()Z
        at avtas.lmcp.lmcpgen.JuliaMethods.BuildDependencyGraph(JuliaMethods.java:63)
        at avtas.lmcp.lmcpgen.JuliaMethods.root_module_includes(JuliaMethods.java:369)
        ... 8 more
```

On the new computer I have 

```
❯ java -version
openjdk version "1.8.0_412"
OpenJDK Runtime Environment Corretto-8.412.08.1 (build 1.8.0_412-b08)
OpenJDK 64-Bit Server VM Corretto-8.412.08.1 (build 25.412-b08, mixed mode)
```

vs the following for the old

```
❯ java -version
openjdk version "1.8.0.2.1" 2022-08-18
OpenJDK Runtime Environment (build 1.8.0.2.1+1-1)
OpenJDK 64-Bit Server VM (build 18.0.2.1+1-1, mixed mode, sharing)
```

Generation works fine for all other template flags. According to  @ctaylor80 seems to be related to this https://stackoverflow.com/questions/55312387/why-my-stringutils-class-doesnt-have-isblank

--- a/src/avtas/lmcp/lmcpgen/JuliaMethods.java
+++ b/src/avtas/lmcp/lmcpgen/JuliaMethods.java
@@ -60,7 +60,7 @@ public class JuliaMethods {
         for (DependencyContainer dc : missingDependencies) {
             StructInfo si = dc.structInfo;
             
-            if (!si.extends_name.isBlank()) { //check dependencies on interfaces
+            if (!si.extends_name.isEmpty()) { //check dependencies on interfaces
                 DependencyContainer extendsStruct = dc.find(missingDependencies, si.extends_name, si.extends_series);
                 dc.dependencies.add(extendsStruct);
                 //System.out.println(String.format("%s depends on %s", si.name, extendsStruct.structInfo.name));
@@ -96,14 +96,11 @@ public class JuliaMethods {
         return ws + st.name;
     }
     
-    public static String extends_check(MDMInfo[] infos, MDMInfo info, final File outfile, StructInfo st, EnumInfo en, String ws) throws Exception {
-        return ws + (st.extends_name.isBlank() ? "LMCP.LmcpBase" : st.extends_name);
-    }
     public static String extends_impl(MDMInfo[] infos, MDMInfo info, final File outfile, StructInfo st, EnumInfo en, String ws) throws Exception {
-        return ws + (st.extends_name.isBlank() ? "LmcpMessage" : st.extends_name);
+        return ws + (st.extends_name.isEmpty() ? "LmcpMessage" : st.extends_name);
     }
     public static String extends_series(MDMInfo[] infos, MDMInfo info, final File outfile, StructInfo st, EnumInfo en, String ws) throws Exception {
-        return ws + (st.extends_name.isBlank() ? "LMCP." : st.extends_series + ".");
+        return ws + (st.extends_name.isEmpty() ? "LMCP." : st.extends_series + ".");
     }
         
     public static String list_imports(MDMInfo[] infos, MDMInfo info, final File outfile, StructInfo st, EnumInfo en, String ws) throws Exception {
@@ -327,7 +324,7 @@ public class JuliaMethods {
             else {
                 str += ws + name + "::" + type;
                 String defaultVal = f.defaultVal;
-                if (defaultVal.isBlank()) {
+                if (defaultVal.isEmpty()) {
                     defaultVal = getDefaultValue(f.type);
                 }
             }
@@ -539,12 +536,12 @@ public class JuliaMethods {
             }
             else if (f.type.equalsIgnoreCase("string")) {
                 str += ws + name + "::" + type;
-                str += " = " + (f.defaultVal.isBlank() ? "\"\"" : "\"" + f.defaultVal + "\"");
+                str += " = " + (f.defaultVal.isEmpty() ? "\"\"" : "\"" + f.defaultVal + "\"");
             }
             else {
                 str += ws + name + "::" + type;
                 String defaultVal = f.defaultVal;
-                if (defaultVal.isBlank()) {
+                if (defaultVal.isEmpty()) {
                     defaultVal = getDefaultValue(f.type);
                 }
                 str += " = " + type + "(" + defaultVal + ")";

